CMakeLists.txt

```diff
@@ -120,11 +138,51 @@ configure_file(${CMAKE_SOURCE_DIR}/src/rajaperf_config.hpp.in
 
 include_directories($<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/bin>)
 
-# Make sure RAJA flag propagate (we need to do some house cleaning to
+# Make sure RAJA flags propagate (we need to do some tidying to
 # remove project-specific CMake variables that are no longer needed)
 set (CUDA_NVCC_FLAGS ${RAJA_NVCC_FLAGS})
 
 #
 # Each directory in the perf suite has its own CMakeLists.txt file.
 #
+
+# The statement below is required for Kokkos compilation.
+if(ENABLE_KOKKOS)
+  include_directories(SYSTEM ${CMAKE_CURRENT_SOURCE_DIR}/tpl/RAJA/include/)
```

Comment:
is this for basic type parameterization that the perf suite inherits from RAJA?

Reply:
We removed the `include_directories(SYSTEM ${CMAKE_CURRENT_SOURCE_DIR}/tpl/RAJA/include/)` line from this bloc.  This line was likely inserted as a hack in the early prototyping stages.